charts/argo-events/ci/test-values.yaml

```diff
@@ -1,6 +1,6 @@
 serviceAccount: argo-events-sa-test
-additionalSaNamespaces:
-  - nsone
-  - nstwo
+# additionalSaNamespaces:
+#  - nsone
+#  - nstwo
```

Comment:
Whats the purpose of this change? Does the new workflow fail?

Reply:
Ahh, I see. The current workflow does not install the chart (eg. on a KinD cluster). And now with the new chart-testing GitHub action, we perform linting AND install tests.